PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/springcloud/deploy/SpringCloudDeploymentState.java

```diff
@@ -173,6 +196,52 @@ protected void updateTelemetryMap(@NotNull Map<String, String> telemetryMap) {
         telemetryMap.putAll(springCloudDeployConfiguration.getModel().getTelemetryProperties());
     }
 
+    private void validateSpringBootApp(String finalJar) throws AzureExecutionException, IOException {
+        final JarFile jarFile = new JarFile(finalJar);
+        final Attributes maniFestAttributes = jarFile.getManifest().getMainAttributes();
+        final String mainClass = maniFestAttributes.getValue(MAIN_CLASS);
+        final String library = maniFestAttributes.getValue(SPRING_BOOT_LIB);
+        if (StringUtils.isEmpty(mainClass)) {
+            throw new AzureExecutionException(MAIN_CLASS_NOT_FOUND);
+        }
+        if (StringUtils.isEmpty(library)) {
+            throw new AzureExecutionException(FAILED_TO_LOAD_DEPENDENCIES);
```

Comment:
FAILED_TO_LOAD_DEPENDENCIES -> Missing Spring-Boot-Lib in ${jarFilePath}!META-INF/MANIFEST.MF, did you use Spring Boot Maven plugin to repackage the target jar?